### PR TITLE
Update fetch results

### PR DESCRIPTION
**Pull request by richardjgowers**
fixes the tarball used for gather command for new Settings

re-enables the tests for gather than I'd previously marked as xfail while we were hashing out the new settings

### CODE REVIEW

**Comment by richardjgowers** [comment]
I've checked locally that the results of `openfe gather` with all report options gives the same results with this file and with the previous release

File: devtools/data/fix_rbfe_results.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        f.add(loc, arcname=os.path.basename(loc))
+
+
+def replace_settings(fn, new_settings):
+    """replace settings instances in *fn* with *new_settings*"""
+    with open(fn, 'r') as f:
+        data = json.load(f)
+
+    for k in data['protocol_result']['data']:
+        # s = data['protocol_result']['data'][k][0]['inputs']['settings']
```
Comment (mikemhenry) [inline devtools/data/fix_rbfe_results.py:33]:
```suggestion
```

File: devtools/data/fix_rbfe_results.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        data['protocol_result']['data'][k][0]['inputs']['settings'] = new_settings
+
+    for k in data['unit_results']:
+        # s = data['unit_results'][k]['inputs']['settings']
```
Comment (mikemhenry) [inline devtools/data/fix_rbfe_results.py:37]:
```suggestion
```

**Review by mikemhenry — APPROVED** [review]
LGTM, just a suggestion to removed some commented out code.

**Pull request merged**